build.sh

```diff
@@ -4,9 +4,18 @@
 set -e
 
 echo "Analyzing with `dartanalyzer --version`"
-dartanalyzer --strong --fatal-warnings lib/*.dart test/*.dart
+dartanalyzer="dartanalyzer --strong --fatal-warnings lib/*.dart test/*.dart"
+if [ "$TRAVIS_DART_VERSION" = "dev" ]; then
+  dartanalyzer="$dartanalyzer --preview-dart-2"
```

Comment:
Just fyi, at this point, `--preview-dart-2` to the analyzer is the same as `--strong` plus supporting optional new and const.